AgniSchema.sql

```diff
@@ -0,0 +1,182 @@
+-- MySQL dump 10.13  Distrib 5.5.46, for debian-linux-gnu (x86_64)
+--
```

Comment:
Looks good! Just a few corrections, and I think we should discuss how we want the Message entity to be structured.